src/app/home-page/widget/news-updates/notification-updates.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-card
```

Comment:
Not sure if others feel the same, but I would prefer it if each notification was in its own mat card/banner so that the separation is clearer. In your video with the two notifications, at first glance, I thought it was a really long notification.

I also think that with separate banners, the dismiss button is more clearly associated with the respective notification. 

Below is what it looks like on smaller screens.

Small window:
![image](https://user-images.githubusercontent.com/25287123/224425931-efaf4263-e2b1-497f-aeff-2c41e697ac16.png)

Phone screen:
![image](https://user-images.githubusercontent.com/25287123/224426023-9438c475-d064-4954-86e3-0a9f5048496f.png)

Reply:
Makes sense 👍️:
![image](https://user-images.githubusercontent.com/97123241/224438903-a8082082-57a4-4f8c-a3a0-c87cde5445ac.png)
![image](https://user-images.githubusercontent.com/97123241/224439176-c83fa9a8-b209-4353-a25d-8a7c7dc4d620.png)